spanner/google/cloud/spanner_v1/gapic/spanner_client.py

```diff
@@ -113,10 +121,22 @@ def __init__(self,
 
         # Create the channel.
         if channel is None:
+            options = None
+
+            if HAS_GRPC_GCP:
+                # Initialize grpc gcp config for spanner api.
```

Comment:
I'd be nice if grpc_gcp had a helper that was just:

```python
grpc_gcp_config = grpc_grpc.config_from_text_pb(...)
```

Reply:
Good point. I'll add this helper in grpc_gcp

Reply:
It's done. Seems like the above link of this file points to an old commit.